client/my-sites/site-logs/main.tsx

```diff
@@ -9,20 +9,21 @@ import { SiteLogsTab, useSiteLogsQuery } from 'calypso/data/hosting/use-site-log
 import { getSelectedSiteId } from 'calypso/state/ui/selectors';
 import { SiteLogsTabPanel } from './components/site-logs-tab-panel';
 import { SiteLogsTable } from './components/site-logs-table';
+import Toolbar from './components/toolbar';
 
 export function SiteLogs() {
 	const { __ } = useI18n();
 	const siteId = useSelector( getSelectedSiteId );
 	const moment = useLocalizedMoment();
 
-	const [ startTime ] = useState( moment().subtract( 7, 'd' ).unix() );
-	const [ endTime ] = useState( moment().unix() );
+	const [ startTime, setStartTime ] = useState( moment().subtract( 7, 'd' ).unix() );
+	const [ endTime, setEndTime ] = useState( moment().unix() );
 
 	const [ logType, setLogType ] = useState< SiteLogsTab >( () => {
 		const queryParam = new URL( window.location.href ).searchParams.get( 'log-type' );
 		return (
 			queryParam && [ 'php', 'web' ].includes( queryParam ) ? queryParam : 'php'
-		) as SiteLogsTab;
+		 ) as SiteLogsTab;
```

Comment:
This rogue space has come back to break the build somehow 😅 Do you have VS Code set up with eslint formatting? p4TIVU-9Lo-p2